doc/man1/openssl-dgst.pod

```diff
@@ -190,6 +182,10 @@ specified.
 When used with the B<-engine> option, it specifies to also use
 engine B<id> for digest operations.
 
+=item B<-rand> I<file>, B<-writerand> I<file>
+
+See L<openssl(1)/Random state options> for more information.
+
```

Comment:
Why was this added?

Reply:
Because it's in the code.  You can do ```./util/find-doc-nits -c | grep dgst``` for example.

Reply:
Let me ask in  a different way; why add a repeat when it's already present, 20 lines up?

Reply:
oh, that.  cut/paste/paste error.  :)   fixed.